### ISSUE TITLE
Please release new versions officially (2)

### ISSUE THREAD

**Issue by lorand-horvath** [nolimits4web/Swiper#5135]
Follow up for https://github.com/nolimits4web/swiper/issues/5134
Is it so difficult to release the new versions? Do you think we have time to explain away such a simple thing? What do you expect, to dumb it down to a 5 year old's level?
I just want to see new versions released/appearing here https://github.com/nolimits4web/swiper/releases
Is that too much to ask? If so, go ahead and close this issue and you can be sure I won't contribute in the future, in any way, shape or form.

**Comment by vltansky** [comment]
@lorand-horvath https://github.com/nolimits4web/swiper/releases/tag/v7.2.0
